### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by storrence88**
- Update spelling to American version of 'Behavior'
- Capitalize RSpec to match the rest of documentation
- Add code ticks to emphasize code portion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
 # RSpec
 
-Behaviour Driven Development for Ruby
+Behavior Driven Development for Ruby
 
 # Description
 
-rspec is a meta-gem, which depends on the
+RSpec is a meta-gem, which depends on the
```
Comment (JonRowe) [inline README.md:7]: ``` `rspec` ``` would be preferred here, as its the gem name, not the code constant.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
 # RSpec
 
-Behaviour Driven Development for Ruby
+Behavior Driven Development for Ruby
```
Comment (JonRowe) [inline README.md:3]: Argh you have me torn between the correct spelling and consistency :joy:

File: README.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
 [rspec-expectations](https://github.com/rspec/rspec-expectations)
 and [rspec-mocks](https://github.com/rspec/rspec-mocks) gems. Each of these
 can be installed separately and loaded in isolation using `require`. Among
 other benefits, this allows you to use rspec-expectations, for example, in
-Test::Unit::TestCase if you happen to prefer that style.
+`Test::Unit::TestCase` if you happen to prefer that style.
 
 Conversely, if you like RSpec's approach to declaring example groups and
-examples (`describe` and `it`) but prefer Test::Unit assertions and
+examples (`describe` and `it`) but prefer `Test::Unit` assertions and
```
Comment (pirj) [inline README.md:16]: Putting Test::Unit in backquotes seems unnecessary, as the authors [refer to it as "Test::Unit" or "test-unit"](https://github.com/test-unit/test-unit/#description).

File: README.md [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
 [mocha](https://github.com/freerange/mocha), [rr](https://github.com/rr/rr)
```
Comment (pirj) [inline README.md:17]: Just for the consistency, do you mind fixing "rr" to "RR", as its project page [uses the latter as a library name](https://github.com/rr/rr#rr---).
Also, ["Mocha"](https://github.com/freerange/mocha#mocha---) and ["FlexMock"](https://github.com/jimweirich/flexmock#flex-mock----making-mocking-easy) (or "Flex Mock"?).

File: README.md [hunk 5]
```diff
@@ -1,19 +1,19 @@
...
-Behaviour Driven Development for Ruby
+Behavior Driven Development for Ruby
 
 # Description
 
-rspec is a meta-gem, which depends on the
+RSpec is a meta-gem, which depends on the
 [rspec-core](https://github.com/rspec/rspec-core),
 [rspec-expectations](https://github.com/rspec/rspec-expectations)
 and [rspec-mocks](https://github.com/rspec/rspec-mocks) gems. Each of these
```
Comment (pirj) [inline README.md:10]: It also makes sense to put those in backquotes, since those are gem names.

File: README.md [hunk 6]
```diff
@@ -1,19 +1,19 @@
...
 can be installed separately and loaded in isolation using `require`. Among
 other benefits, this allows you to use rspec-expectations, for example, in
```
Comment (pirj) [inline README.md:12]: Also here `rspec-expectations`.

**Review by pirj — APPROVED** [review]
Thanks!
A couple of notes to improve consistency.

File: README.md (hunk 3 above)
Comment (JonRowe) [reply README.md:16]: I disagree, our convention is to use backticks when we have code names.

**Comment by pirj** [comment]
Closing as stale. Please feel free to reopen if you're up to address the remaining notes.

**Pull request closed without merging**